ROBOT_NAMESPACE = "capo"

TOPIC_NAME_CAPO_TICKER = "ticker"
TICKER_NODE_NAME = "ticker"
CAPO_TICKER_DEFAULT_RATE_HZ = 10
ROS_LAUNCH_PARAM_NAME_TICKER_PUBLISH_RATE = "/%s/%s/ticker_publish_rate_hz" % (ROBOT_NAMESPACE, TICKER_NODE_NAME)


CAMERA_NODE_NAME = "capo2_camera_publisher"
CAMERA_DEFAULT_ID = 1
ROS_LAUNCH_PARAM_CAMERA_ID = "/%s/camera_id" % CAMERA_NODE_NAME
ROS_LAUNCH_PARAM_CAMERA_WIDTH = "/%s/camera_width" % CAMERA_NODE_NAME
ROS_LAUNCH_PARAM_CAMERA_HEIGHT = "/%s/camera_height" % CAMERA_NODE_NAME
